src/components/Comments.jsx

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from "react";
+import { getComments } from "../data";
+
+const Comments = ({ article_id }) => {
+  const [comments, setComments] = useState([]);
+  const [IsLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getComments(article_id).then((commentsAPI) => {
+      setComments(commentsAPI);
+      setIsLoading(false);
+    });
+  }, []);
+  return (
+    <div>
+      <h3>comments</h3>
+      {comments.map((comment) => {
```

Comment:
- [ ] Could you add some functionality so that the UI should reasonably account for an article that has no comments yet?